CONTRIBUTING.md

```diff
@@ -3,3 +3,9 @@
 Thanks for taking the time to contribute! 😃 🚀
 
 Please refer to our [Contributing Guide](https://traceloop.com/docs/openllmetry/contributing/overview) for instructions on how to contribute.
+
+## 💫 Contributors
```

Comment:
Can you move this to the readme?

Reply:
sir, it's done. Are there any other changes needed?

Reply:
Yes @kushal34712 see comment above: 

> Can you not remove the instructions on the PR template , and update the title accordingly? (Should be "docs: ..."

Reply:
@nirga sir its done